Source/JavaScriptCore/API/glib/JSCContextPrivate.h

```diff
@@ -19,8 +19,6 @@
 
 #pragma once
 
-#include "JSCContext.h"
-#include "JSCValue.h"
```

Comment:
And this? I think it's ok to include instead of forward declare in these private headers.

Reply:
This was a sleepy fix for the same missing include path as above.